package.json

```diff
@@ -1013,13 +1014,13 @@
     "base64-js": "^1.3.1",
     "bitmap-sdf": "^1.0.3",
     "blurhash": "^2.0.1",
+    "borc": "3.0.0",
```

Comment:
This package doesn't appear to be maintained.  It's an older fork of https://github.com/hildjj/node-cbor that is active, was this considered?

Reply:
@jbudz unfortunately the usage of `cbor-x` spread along our use cases too fast and it doesn't seem to have a standard return type equal to the one of node-cbor (mostly around Buffer vs uint8), so I found that `borc` that followed a similar approach to `cbor-x` and it worked instantly. My thought was that the teams can work on `@kbn/cbor` to replace it later if they needed.

Reply:
thanks for the info, I'll leave it for the associated teams reviewing then